build.sh

```diff
@@ -97,6 +98,12 @@ RUN_UT=
 WITH_GCOV=OFF
 WITH_HDFS=ON
 
+
+# Disable WITH_HDFS on aarch64
+if [[ "${MACHINE_TYPE}" == "aarch64" ]]; then
+    WITH_HDFS=OFF
```

Comment:
Why disable HDFS?

Reply:
Disabling the HDFS bypasses the building error `File in wrong format libhdfs`. To point it out, the PR description is updated.

Reply:
I think the reason is that the JDK we download is for x86 platform. For aarch64, we should download the corresponding one.

Reply:
Seems that it's true. I've revert this and will fix this problem in next PR.